pytest_mozwebqa/pytest_mozwebqa.py

```diff
@@ -199,13 +192,6 @@ def pytest_addoption(parser):
     config = ConfigParser.ConfigParser(defaults={'baseurl': ''})
     config.read('mozwebqa.cfg')
```

Comment:
It's intentional, for now. Before the next release I expect to move everything into `setup.cfg`.